src/main/java/mate/academy/bookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,35 @@
 package mate.academy.bookstore;
 
+import java.math.BigDecimal;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.service.BookService;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    private final BookService bookService;
+
+    public OnlineBookStoreApplication(BookService bookService) {
+        this.bookService = bookService;
+    }
```

Comment:
it is a bad idea to make dependencies for the main class

you can move this service as a dependency of a `CommandLineRunner` bean:
```
@Bean
    public CommandLineRunner commandLineRunner(BookService bookService) {
....
```